### PR TITLE
1442 image gallery tag changes

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: bindings/java/src/org/sleuthkit/datamodel/TagName.java [hunk 1]
```diff
@@ -33,39 +33,45 @@
...
+		YELLOW	("Yellow",	"#FFFF00"), //NON-NLS
+		OLIVE	("Olive",	"#808000"), //NON-NLS
+		LIME	("Lime",	"#00FF00"), //NON-NLS
+		GREEN	("Green",	"#008000"), //NON-NLS
+		AQUA	("Aqua",	"#00FFFF"), //NON-NLS
+		TEAL	("Teal",	"#008080"), //NON-NLS
+		BLUE	("Blue",	"#0000FF"), //NON-NLS
+		NAVY	("Navy",	"#000080"), //NON-NLS
+		FUCHSIA	("Fuchsia", "#FF00FF"), //NON-NLS
+		PURPLE	("Purple",	"#800080"); //NON-NLS
```
Comment (rcordovano) [inline bindings/java/src/org/sleuthkit/datamodel/TagName.java:52]: What do the '#' chars signify in these strings?

File: bindings/java/src/org/sleuthkit/datamodel/TagName.java [hunk 2]
```diff
@@ -33,39 +33,45 @@
...
 			this.name = name;
 		}
 
 		String getName() {
 			return name;
 		}
+		
+		public String getHexColorCode() {
+			return hexString;
+		}
```
Comment (rcordovano) [inline bindings/java/src/org/sleuthkit/datamodel/TagName.java:74]: Would it make more sense to a client if this were named getRgbValue with a comment that the return value was in hex format?

Comment (kellykelly3) [reply bindings/java/src/org/sleuthkit/datamodel/TagName.java:74]: That is a better name.

File: bindings/java/src/org/sleuthkit/datamodel/TagName.java (hunk 1 above)
Comment (kellykelly3) [reply bindings/java/src/org/sleuthkit/datamodel/TagName.java:52]: The # is part of the hex value.  I think it has something to do with the HTML specification?  I quick google, didn't tell me why the # is there other than its expected by the <style> tag.

Comment (rcordovano) [reply bindings/java/src/org/sleuthkit/datamodel/TagName.java:52]: It sounds like the '#' character is HTML-markup specific, then. I imagine you are trimming it off in IG code I haven't seen yet. I think we could probably drop it from the enum, but I don't think it much matters.

Comment (kellykelly3) [reply bindings/java/src/org/sleuthkit/datamodel/TagName.java:52]: Nope.  I don't trim it.  The color methods in both java se and javafx take it with the #.  In fact if we didn't have it on there I think I would have to add it, maybe.

**Pull request merged**